lib/browser/init.ts

```diff
@@ -106,7 +106,7 @@ if (process.resourcesPath) {
   for (packagePath of searchPaths) {
     try {
       packagePath = path.join(process.resourcesPath, packagePath)
-      packageJson = __non_webpack_require__(path.join(packagePath, 'package.json')) // eslint-disable-line
+      packageJson = Module._load(path.join(packagePath, 'package.json'))
```

Comment:
this seems unrelated to me...?

Reply:
`__non_webpack_require__` maps to the global node provided require function, now we are an install module that require function is `NativeModule.require` which does not load non-internal code.  We need to explicitly load non-internal code so we do what node does internally and directly call `Module._load`